tshirts/tshirts.cs

```diff
@@ -1,19 +1,28 @@
-﻿using System;
+using System;
 using System.Diagnostics;
 
-namespace TshirtSpace {
-    class Tshirt {
-        static string Size(int cms) {
-            if(cms < 38) {
-                return "S";
-            } else if(cms > 38 && cms < 42) {
-                return "M";
-            } else {
-                return "L";
-            }
-        }
+namespace TshirtSpace
+{
+    class Tshirt
+  {
+    static string Size(int cms)
+    {
+      if (cms < 38)
+      {
+        return "S";
+      }
+      else if (cms > 38 && cms < 42)
+      {
+        return "M";
+      }
+      else
+      {
+        return "L";
+      }
+    }
         static void Main(string[] args) {
             Debug.Assert(Size(37) == "S");
+            Debug.Assert(Size(38) == "S");
             Debug.Assert(Size(40) == "M");
             Debug.Assert(Size(43) == "L");

```

Comment:
Boundary value of 42 needs to be asserted too.

Reply:
Boundary value of 42 is added in the test